src/player.html

```diff
@@ -69,16 +69,28 @@ <h2>{{instance.name}}</h2>
 									</md-select>
 								</md-input-container>
 							</md-card-actions>
-
-							<md-card-actions layout layout-align="start start" ng-if="question.options.displayStyle == verticalList">
+							<!-- Check All That Apply -->
+							<md-card-actions layout="column" layout-align="start start" ng-if="question.options.displayStyle == verticalList && question.options.questionType == 'check-all-that-apply'">
+								<div layout="row">
+									<span class="response-limit-description" ng-if="question.options.minResponseLimit">Select at least {{question.options.minResponseLimit}} item{{ question.options.minResponseLimit > 1 ? "s" : "" }}.</span>
+									<span class="response-limit-description" ng-if="question.options.maxResponseLimit">Select at most {{question.options.maxResponseLimit}} item{{ question.options.maxResponseLimit > 1 ? "s" : "" }}.</span>
+								</div>
 								<md-input-container layout="column">
 									<md-checkbox
 										ng-repeat="answer in question.answers"
 										ng-model="responses[$parent.$index][$index]"
 										ng-init="responses[$parent.$index][$index] = false"
-										ng-change="updateCompleted()">
+										ng-change="updateCheckAllThatApply($parent.$index, $index)">
 										{{answer.text}}									
 									</md-checkbox>
+									<md-checkbox
```

Comment:
Might suggest altering the coloration of this somehow to indicate that it behaves differently from the rest - in the event that the 'None of the Above' option doesn't actually read 'None of the Above' it isn't immediately clear why clicking it automatically un-checks all of the other options.